### PR TITLE
contrib: add shibuqam to extract information from a shibboleth authentication

### PR DESCRIPTION
**Pull request by privat**

### CODE REVIEW

File: contrib/shibuqam/shibuqam.nit [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# The main method to use is `HttpRequest::uuser` and it extracts the informatio
+# of the authenticated user from the request header.
+# The real authentication must be done by a mandatory reverse proxy server.
+module shibuqam
+
+import nitcorn
+private import md5
+
+# Information on a user from Shibboleth/UQAM
+class UUser
```
Comment (Morriar) [inline contrib/shibuqam/shibuqam.nit]: Please don't...

Comment (Morriar) [inline contrib/shibuqam/shibuqam.nit]: Please just...

Comment (Morriar) [inline contrib/shibuqam/shibuqam.nit]: No... it's wrong...

Comment (privat) [inline contrib/shibuqam/shibuqam.nit]: :smiling_imp:

File: contrib/shibuqam/shibuqam.nit [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Gather the authenticated users on UQAM websites.
+#
+# The main method to use is `HttpRequest::uuser` and it extracts the informatio
```
Comment (R4PaSs) [inline contrib/shibuqam/shibuqam.nit]: :s/informatio/information

File: contrib/shibuqam/shibuqam.nit [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+# of the authenticated user from the request header.
+# The real authentication must be done by a mandatory reverse proxy server.
+module shibuqam
+
+import nitcorn
+private import md5
+
+# Information on a user from Shibboleth/UQAM
+class UUser
+	# The *code permanant* (or the uid for non student)
```
Comment (Morriar) [inline contrib/shibuqam/shibuqam.nit]: `permanent`

File: contrib/shibuqam/shibuqam.nit (hunk 1 above)
Comment (privat) [inline contrib/shibuqam/shibuqam.nit]: choose a better name if you have one

Comment (R4PaSs) [inline contrib/shibuqam/shibuqam.nit]: Why not `User` ? Name clash ? If so, qualified names should solve the issue right ? Or am I missing something ?

**Comment by Morriar** [comment]
+1 beside the name

**Comment by privat** [comment]
reroll: rename UUser to User + spell

**Comment by R4PaSs** [comment]
+1!

**Pull request merged**